opcua/common/subscription.py

```diff
@@ -208,39 +208,58 @@ def _get_node(self, nodeid):
             node = Node(self.server, ua.NodeId(nodeid))
         return node
 
-    def _get_filter_from_event_type(self, eventtype):
-        eventtype = self._get_node(eventtype)
+    def _get_filter_from_event_type(self, eventtypes):
+        # FIXME must be a nicer way to make a list out of eventtypes when it's passed
+        # FIXME an integer (such as ua.ObjectIds.BaseEventType)
```

Comment:
I do not think there is a better way. but add support for tuples with 'type(xx) in (list, tuple):'